lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -26,9 +26,26 @@ module.exports = [
         details: {
           overallSavingsBytes: '>45000',
           overallSavingsMs: '>500',
-          items: {
-            length: 1,
-          },
+          items: [
+            {
+              url: 'http://localhost:10200/byte-efficiency/script.js',
+              totalBytes: 53181,
+              wastedBytes: 46481,
+              wastedPercent: 87.40117365133875,
```

Comment:
I would feel slightly better about a looser assertion here

Reply:
wb a new syntax like `x % 5` to indicate within +/- 5% of x?

Reply:
Oooh I love that, how about `x +/- y` though ;)

Reply:
+/- landed right we can update these now?